OrbitGl/TimeGraph.cpp

```diff
@@ -726,41 +720,34 @@ void TimeGraph::SortTracks() {
   }
 }
 
-//----------------------------------------------------------------------------
-void TimeGraph::OnLeft() {
+const TextBox* TimeGraph::FindPrevious() {
   TextBox* selection = Capture::GSelectedTextBox;
+  const TextBox* previous = nullptr;
```

Comment:
Consider adding direct return statements. That'd usually easier to read. 

That is: 
```
  if (selection) {
    const Timer& timer = selection->GetTimer();
    if (timer.m_Type == Timer::GPU_ACTIVITY) {
      return GetOrCreateGpuTrack(GetGpuTimelineHash(timer))->GetLeft(selection);
    } else {
      return GetOrCreateThreadTrack(timer.m_TID)->GetLeft(selection);
    }
  }
  return nullptr;
```

Reply:
This also makes it clear that you are returning a nullptr when you fall through the cases. Not that obvious with the current code.

Reply:
You are clearly right